### ISSUE TITLE
Selecting which user to use for interactions

### ISSUE THREAD

**Issue by bitboxer** [metabolist/metatext#149]
If i have several accounts i would love to have the possibility to choose which account to use for boosting and replying to a post and following an account.

**Comment by eggdropsoap** [comment]
If you long-press on an interaction button under a post, it pops down an account chooser menu.

**Comment by bitboxer** [comment]
Ah, right. But this does not work for the follow button, or is there a hidden way to do it, too?

**Comment by eggdropsoap** [comment]
You’re right, it does not work for following! I just tried and confirmed it doesn’t.

I also don’t see another way it’s done, so consider this an endorsement of the feature request.

It might be worth editing the request itself to focus on adding that functionality to the follow button, so that it’s consistent with other action buttons.

**Comment by rstockm** [comment]
+1 for this feature - I love the long-press account-switcher option on the interaction buttons  and would love to see it in follow, too.
